src/wdl/workflows/RegisterEGAExperimentAndRun/RegisterEGAExperimentAndRun.wdl

```diff
@@ -182,15 +173,13 @@ task CheckEGAFileValidationStatus {
 task DeleteFileFromBucket {
     input {
         String aggregation_path
-        String aggregation_index_path
     }
 
     String aggregation_md5_path  = aggregation_path + ".md5"
 
     command <<<
         set -eo pipefail
         gsutil rm -a ~{aggregation_path}
-        gsutil rm -a ~{aggregation_index_path}
```

Comment:
Won't this leave the index in the directory?

Reply:
We can just generate the index path using the aggregation_path, that way we don't have to change the metadata/motorcade automation again. I think we still want to delete the index path though if we're also deleting the aggregation